src/components/TextInput/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Input,
+  Container,
+  ContainerError,
+  TextError,
+} from './styles';
+
+import {Label } from '../Label'
+import { Colors } from '../../utils/Colors';
+
+export const TextInput = ({
+  isPassword = true,
+  color,
+  placeholder,
+  label,
+  active,
+  errorInput = false, 
+  msgError = "",
+  ...rest }) => {
```

Comment:
Todos as suas funções e métodos devem ter o tipo de retorno.
```suggestion
export const TextInput = ({
  isPassword = true,
  color,
  placeholder,
  label,
  active,
  errorInput = false, 
  msgError = "",
  ...rest }): ReactElement => {
```